src/components/navbar/NavbarListItems.tsx

```diff
@@ -21,13 +21,13 @@ export const mainNavbarItems: mainNavbarItemsType = [
   {
     id: 1,
     icon: <PeopleIcon />,
-    label: "Courses",
+    label: "Communities",
     route: "/",
   },
   {
     id: 2,
     icon: <QuestionMarkIcon />,
-    label: "Course Page",
-    route: "/course",
+    label: "Community 1",
+    route: "/community/1",
```

Comment:
Not sure if you can adjust this so the number is already a variable and not hard-coded?

Reply:
I made this so it easier to develop and test at the moment. In the future it should be removed.